### DISCUSSION TITLE
How to browse links within properties of a note?

### DISCUSSION THREAD

**Discussion by camfisk** [chrisgrieser/shimmering-obsidian#165] [Q&A] [answered]
I have many notes that contain links within their properties. Is there a way I can browse and open these links using this or another feature?: [ʟɪɴᴋ](https://github.com/chrisgrieser/shimmering-obsidian/blob/main/docs/Alfred-based%20Quick%20Switcher.md#browse-links-of-a-note)

**Comment by chrisgrieser** [comment, marked as answer]
The workflow uses the links that the [metadata extractor plugin](https://obsidian.md/plugins?id=metadata-extractor) provides. I'd suggest opening a feature request at that plugin's repo, then you should also be able to browse those links from Alfred

**Reply by camfisk** [reply]
Seems like that plugin is abandoned. 🙁
